scripts/test_schemas.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator=$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)
+docker pull onsdigital/eq-schema-validator:dropdown-schemas
+validator=$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator:dropdown-schemas)
```

Comment:
We'll need to remember to change this when the validator pull request merges

Reply:
@MebinAbraham this can be changed back now as the validator change is merged

Reply:
@jonnyshaw89 Done